src/utils.test.ts

```diff
@@ -230,20 +233,44 @@ describe('getPlaywrightInstance', () => {
 
     jest.doMock('playwright', () => ({
       firefox: 'firefox',
+      chromium: 'chromium',
     }))
 
     const { instance } = getPlaywrightInstance('playwright', 'firefox')
     expect(instance).toEqual('firefox')
   })
 
-  it('should return specified instance from specified playwright package', async () => {
-    spy.mockResolvedValue('chromium')
+  it('should return specified instance from specified playwright package', () => {
+    spy.mockResolvedValue({
+      chromium: 'chromium',
+    })
 
     jest.doMock('playwright-chromium', () => ({
       chromium: 'chromium',
     }))
 
-    const { instance } = getPlaywrightInstance('chromium', 'chromium')
+    const { instance } = getPlaywrightInstance(
+      { chromium: 'chromium' },
+      'chromium',
+    )
     expect(instance).toEqual('chromium')
   })
+
+  it('should throw error when playwright package is not provided', () => {
+    spy.mockResolvedValue({
+      chromium: 'chromium',
+    })
+
+    jest.doMock('playwright-chromium', () => ({
+      chromium: 'chromium',
+    }))
+
+    let error
+    try {
+      getPlaywrightInstance({ chromium: 'chromium' }, 'firefox')
```

Comment:
toThrow would be nicer: https://jestjs.io/docs/en/expect#tothrowerror

Also compare it with the error which is exported instead of creating your own error again.

Reply:
@mxschmitt youp!  good catch!